### PR TITLE
Dasharo value proposition

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

File: docs/osf-trivia-list/dasharo.md [hunk 1]
```diff
@@ -154,3 +154,83 @@ Dasharo is not coreboot firmware distribution because we also may imagine
 Dasharo pure EDKII, oreboot or based on any other open-source firmware
 framework. Dasharo typically is based on coreboot and EDKII, plain coreboot is
 not enough in most cases for booting modern computer.
+
+## What value Dasharo provide in comparison to coreboot?
```
Comment (macpijan) [inline docs/osf-trivia-list/dasharo.md:158]:
```suggestion
## What value Dasharo provides in comparison to coreboot?
```

File: docs/osf-trivia-list/dasharo.md [hunk 2]
```diff
@@ -154,3 +154,83 @@ Dasharo is not coreboot firmware distribution because we also may imagine
...
+  some level of support always would be available to the community.
+
+### Dasharo Transparent Validation
+
+* We provide transparent validation results - coreboot in itself provide no
+  guarantees around release quality and do not provide binary distribution (for
+  reference please check [coreboot project scope][coreboot-scope], we provide
+  those in scope of validation we perform.
+
+### Dasharo Trustworthiness for All
```
Comment (macpijan) [inline docs/osf-trivia-list/dasharo.md:233]: We define the same term here: https://github.com/Dasharo/docs/pull/507/files#diff-c8be10a4380efbce4c82013c5d1e8d7d42423eda7107a96990f0c54284767b2cR307
But in different manner

File: docs/osf-trivia-list/dasharo.md [hunk 3]
```diff
@@ -154,3 +154,83 @@ Dasharo is not coreboot firmware distribution because we also may imagine
...
+  cost-effectiveness, security, transparency and customizability than
+  proprietary alternative.
+* There are seven characterestics of Dasharo:
+    - [zero-touch initial deployment](#dasharo-zero-touch-initial-deployment),
+    - [clean and simple code](#dasharo-clean-and-simple-code),
+    - [long-term maintenance](#dasharo-long-term-maintenance),
+    - [professional support](#dasharo-support-package),
+    - [transparent validation](#dasharo-transparent-validation),
+    - [privacy-respecting implementation](#dashro-privacy-respecting-implementation),
+    - [liberty for the owners](#dasharo-liberty-for-the-owners),
```
Comment (macpijan) [inline docs/osf-trivia-list/dasharo.md:174]: This is mentioned and linked here, but not introduced in this MR.
OTOH, it somehow described in another MR: https://github.com/Dasharo/docs/pull/507/files#diff-c8be10a4380efbce4c82013c5d1e8d7d42423eda7107a96990f0c54284767b2cR301

File: docs/osf-trivia-list/dasharo.md [hunk 4]
```diff
@@ -154,3 +154,83 @@ Dasharo is not coreboot firmware distribution because we also may imagine
...
+    - [trustworthiness for all](#dasharo-trustworthiness-for-all).
+
+### Dasharo Zero-Touch Initial Deployment
+
+Documentation supported hardware provide information about initial deployment,
```
Comment (macpijan) [inline docs/osf-trivia-list/dasharo.md:179]:
```suggestion
Documentation supported hardware provides information about initial deployment,
```

File: docs/osf-trivia-list/dasharo.md [hunk 5]
```diff
@@ -154,3 +154,83 @@ Dasharo is not coreboot firmware distribution because we also may imagine
...
+updates and recovery procedures. Developed by Dasharo Team [Dasharo Tools Suite
+(DTS)][dtsos] operating system support users by automating the deployment
+process, which help reducing errors and inconsistencies, and make sure the
```
Comment (macpijan) [inline docs/osf-trivia-list/dasharo.md:182]:
```suggestion
process, which helps reducing errors and inconsistencies, and make sure the
```

File: docs/osf-trivia-list/dasharo.md [hunk 6]
```diff
@@ -154,3 +154,83 @@ Dasharo is not coreboot firmware distribution because we also may imagine
...
+    - [transparent validation](#dasharo-transparent-validation),
+    - [privacy-respecting implementation](#dashro-privacy-respecting-implementation),
+    - [liberty for the owners](#dasharo-liberty-for-the-owners),
+    - [trustworthiness for all](#dasharo-trustworthiness-for-all).
+
+### Dasharo Zero-Touch Initial Deployment
+
+Documentation supported hardware provide information about initial deployment,
+updates and recovery procedures. Developed by Dasharo Team [Dasharo Tools Suite
+(DTS)][dtsos] operating system support users by automating the deployment
```
Comment (macpijan) [inline docs/osf-trivia-list/dasharo.md:181]: This sentence sounds weird. Why not start it with the subject? We can skip `Developed by Dasharo Team` if it makes it more complex

File: docs/osf-trivia-list/dasharo.md [hunk 7]
```diff
@@ -154,3 +154,83 @@ Dasharo is not coreboot firmware distribution because we also may imagine
...
+  platforms which are moved to branch in coreboot.
+* Firmware update - we are registered [consultants for fwupd/LVFS][lvfs] and
+  enable customers and community platforms, so they can get seamless firmware
+  update in Linux.
+
+### Dasharo Support Package
+
+* Support - Dasharo Team is paid to provide support in the community through
+  [Dasharo Matrix Space][dasharo-matrix], of course we want to introduce some
+  commercial products related to the support to make the business feasible, but
```
Comment (macpijan) [inline docs/osf-trivia-list/dasharo.md:223]: We have such product, and it is named exactly as the name of this section: `Dasharo Support Package`. What is described here, is not DSP.

File: docs/osf-trivia-list/dasharo.md [hunk 8]
```diff
@@ -154,3 +154,83 @@ Dasharo is not coreboot firmware distribution because we also may imagine
...
+  work before those will go upstrea. If ever, we are committed to maintain
+  platforms which are moved to branch in coreboot.
+* Firmware update - we are registered [consultants for fwupd/LVFS][lvfs] and
+  enable customers and community platforms, so they can get seamless firmware
+  update in Linux.
+
+### Dasharo Support Package
+
+* Support - Dasharo Team is paid to provide support in the community through
+  [Dasharo Matrix Space][dasharo-matrix], of course we want to introduce some
```
Comment (macpijan) [inline docs/osf-trivia-list/dasharo.md:222]: Missing URL

File: docs/osf-trivia-list/dasharo.md [hunk 9]
```diff
@@ -154,3 +154,83 @@ Dasharo is not coreboot firmware distribution because we also may imagine
...
+  commercial products related to the support to make the business feasible, but
+  some level of support always would be available to the community.
+
+### Dasharo Transparent Validation
+
+* We provide transparent validation results - coreboot in itself provide no
+  guarantees around release quality and do not provide binary distribution (for
+  reference please check [coreboot project scope][coreboot-scope], we provide
```
Comment (macpijan) [inline docs/osf-trivia-list/dasharo.md:230]: Missing URL

File: docs/osf-trivia-list/dasharo.md [hunk 10]
```diff
@@ -154,3 +154,83 @@ Dasharo is not coreboot firmware distribution because we also may imagine
...
+
+### Dasharo Clean and Simple Code
+
+* Each supported platform has its own branches in Dasharo's coreboot repository:
+    - `<board_name>/develop`, e.g. `msi_ms7d25/develop`, used for bug fixes and
+    development of new features.
+    - `<board_name>/release`, e.g. `msi_ms7d25/release`, contains only the
+    released and validated code. It is used to build the firmware binaries
+    distributed to the community through Dasharo releases.
+* Each supported platform has a build scripts included in Dasharo's coreboot
```
Comment (macpijan) [inline docs/osf-trivia-list/dasharo.md:205]: This is not true for **each** platform

File: docs/osf-trivia-list/dasharo.md [hunk 11]
```diff
@@ -154,3 +154,83 @@ Dasharo is not coreboot firmware distribution because we also may imagine
...
+  Debian/Ubuntu/RedHat and Linux. That means it can provide better
+  cost-effectiveness, security, transparency and customizability than
+  proprietary alternative.
+* There are seven characterestics of Dasharo:
+    - [zero-touch initial deployment](#dasharo-zero-touch-initial-deployment),
+    - [clean and simple code](#dasharo-clean-and-simple-code),
+    - [long-term maintenance](#dasharo-long-term-maintenance),
+    - [professional support](#dasharo-support-package),
+    - [transparent validation](#dasharo-transparent-validation),
+    - [privacy-respecting implementation](#dashro-privacy-respecting-implementation),
```
[the remark this answers is not in the document]
Comment (pietrushnic) [reply docs/osf-trivia-list/dasharo.md]: @macpijan I did rebase and provided correct references to future work: https://github.com/Dasharo/docs/pull/504/commits/a51ca4d68bd9f339f71f23e3edc75f34c21f3018

File: docs/osf-trivia-list/dasharo.md (hunk 3 above)
Comment (pietrushnic) [reply docs/osf-trivia-list/dasharo.md]: a/a https://github.com/Dasharo/docs/pull/504/commits/a51ca4d68bd9f339f71f23e3edc75f34c21f3018

File: docs/osf-trivia-list/dasharo.md [hunk 12]
```diff
@@ -154,3 +154,83 @@ Dasharo is not coreboot firmware distribution because we also may imagine
...
+Use of DTS largely improve firmware adoption, hardware compatibility reporting
+and binary blobs transmission, as well as recovery.
+
+For more details about zero-touch initial deployment please read relevant
+Dasharo Tools Suite
+[documentation](../../dasharo-tools-suite/documentation/#dasharo-zero-touch-initial-deployment).
+
+### Dasharo Clean and Simple Code
+
+* Each supported platform has its own branches in Dasharo's coreboot repository:
```
[the remark this answers is not in the document]
Comment (pietrushnic) [reply docs/osf-trivia-list/dasharo.md]: @macpijan fixed: https://github.com/Dasharo/docs/pull/504/commits/be23e8794b67b7b1d60c0753e075376388561f21

File: docs/osf-trivia-list/dasharo.md (hunk 2 above)
Comment (pietrushnic) [reply docs/osf-trivia-list/dasharo.md:258]: Fixed: https://github.com/Dasharo/docs/commit/a51ca4d68bd9f339f71f23e3edc75f34c21f3018

File: docs/osf-trivia-list/dasharo.md (hunk 8 above)
Comment (pietrushnic) [reply docs/osf-trivia-list/dasharo.md]: Fixed: https://github.com/Dasharo/docs/pull/504/commits/6a8894af3401e8420c1c5ec639b5b504b3b93b0e

**Pull request merged**